source/features/show-whitespace.css

```
@@ -1,18 +1,17 @@
 [data-rgh-whitespace] {
-	position: relative;
 	line-height: 1em;
+	white-space: break-spaces;
+	background-clip: border-box;
+	background-repeat: repeat-x;
+	background-position: left center;
 }
 
-[data-rgh-whitespace]::before {
-	content: attr(data-rgh-whitespace);
-	pointer-events: none;
-	user-select: none;
-	position: absolute;
-	left: 0;
-	top: 0;
-	opacity: 25%;
+[data-rgh-whitespace='tab'] {
+	background-image: url('data:image/svg+xml,%3Csvg preserveAspectRatio="xMinYMid meet" viewBox="0 0 12 24" xmlns="http://www.w3.org/2000/svg"%3E%3Cpath d="M9.5 10.44L6.62 8.12L7.32 7.26L12.04 11V11.44L7.28 14.9L6.62 13.9L9.48 11.78H0V10.44H9.5Z" fill="%2324292E" fill-opacity="0.5"/%3E%3C/svg%3E"');
```

```suggestion
	background-image: url('data:image/svg+xml,%3Csvg preserveAspectRatio="xMinYMid meet" viewBox="0 0 12 24" d="M9.5 10.44L6.62 8.12L7.32 7.26L12.04 11V11.44L7.28 14.9L6.62 13.9L9.48 11.78H0V10.44H9.5Z" fill="%2324292E" fill-opacity="0.5"/%3E%3C/svg%3E"');
```

Also, does `fill` support `rgba()`? Let's use that instead of `fill` + `fill-opacity`

---

`fill` does support `rgba()`, but removing `xmlns` doesn't work as this image essentially renders outside of main HTML context and needs its own context.

---

Also let's reduce the opacity compensating thickness. WDYT?